.github/workflows/patches/20-vvc-werror.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/libavcodec/vvc/Makefile b/libavcodec/vvc/Makefile
+index 6a28d32bc2..dc631897d4 100644
+--- a/libavcodec/vvc/Makefile
++++ b/libavcodec/vvc/Makefile
+@@ -15,3 +15,18 @@ OBJS-$(CONFIG_VVC_DECODER)          +=  vvc/dec.o           \
+                                         vvc/ps.o            \
+                                         vvc/refs.o          \
+                                         vvc/thread.o        \
++
++$(SUBDIR)vvc/dec.o: CFLAGS += -Werror
```

Comment:
We may split or rename files. 
how about

`$(SUBDIR)vvc/*.o: CFLAGS += -Werror`

Reply:
Done in 723a9f02e25b5891d09cf6de78812b32c2347182.